src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -82,24 +83,28 @@ public void testDoGet_FakeUser() throws IOException, ServletException {
     Mockito.verify(mockResponse).sendRedirect("/login");
   }
 
-/*
+
+  /** Tests that when user is a valid admin to forward the request into ./admin */
   @Test
-  public void testDoGet_ValidUser() throws IOException, ServletException {
+  public void testDoGet_ValidAdminUser() throws IOException, ServletException {
     User admin_user =
         new User(
             UUID.randomUUID(),
             "admin_username",
             "admin_password",
-            Instant.now());
+            Instant.now(),
+            true);
 
     Mockito.when(mockSession.getAttribute("user")).thenReturn("admin_username");
     Mockito.when(mockUserStore.getUser("admin_username")).thenReturn(admin_user);
 
+    boolean b = admin_user.isAdmin();
+    Assert.assertEquals(b, true);
+
```

Comment:
I think it's assertEquals (expected Object, actual Object), so would be Assert.assertEquals(true, b) since we're expecting true, so maybe just swap the order?

Reply:
Fixed!